### PR TITLE
2nd Assignment by 202403768 (Seohee Lee)

### PR DESCRIPTION
**Pull request by SH-hufs**
# Assignment Submission

## Student Information
- **Student ID**: 202403768
- **Name**: Seohee Lee
- **Assignment**: 2

## Changed Files

```
[FILE_TREE_PLACEHOLDER]
```

### Problem 1
- [x] Implemented required functionality
- [x] All tests pass locally

### Problem 2
- [x] Implemented required functionality
- [x] All tests pass locally

### Problem 3
- [x] Implemented required functionality
- [x] All tests pass locally

### Problem 4 (Optional)
- [x] Implemented required functionality
- [x] All tests pass locally

### Problem 5 (Optional)
- [x] Implemented required functionality
- [x] All tests pass locally

### Problem 6 (Optional)
- [x] Implemented required functionality
- [x] All tests pass locally

## Testing
- [ ] I have run `python grader.py` locally
- [x] All implemented problems pass their tests
- [ ] I have tested edge cases mentioned in the problem statements

## Additional Notes

---
**Note**: This PR will be automatically validated for:
- Correct directory structure
- Valid Python syntax
- PR title format: `Nth Assignment by {student_id} (Full Name)`
- File organization compliance

### CODE REVIEW

**Comment by hist0613** [comment]
본문에 정보 채워주세요~

File: submissions/202403768/assignment2/problem5/cacheckit/__init__.py
Comment (hist0613) [inline submissions/202403768/assignment2/problem5/cacheckit/__init__.py:1]: 디렉토리 `cacheckit` 이 아니라 `cachekit` 입니다!

File: submissions/202403768/assignment2/problem4/dsops/stats/labels.py [hunk 1]
```diff
@@ -0,0 +1,9 @@
+from collections import Counter
+from typing import List, Dict
+
+
+from collections import Counter
+from typing import List, Dict
+
```
Comment (hist0613) [inline submissions/202403768/assignment2/problem4/dsops/stats/labels.py:7]: import 가 중복이네요~

**Review by hist0613 — CHANGES_REQUESTED** [review]
구현은 잘 하셨습니다 (grader 기준 만점).
리뷰 반영해주시고, credit 제대로 해주세요~
